spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/JGitEnvironmentRepository.java

```diff
@@ -170,13 +195,19 @@ public void afterPropertiesSet() throws Exception {
 	/**
 	 * Get the working directory ready.
 	 */
-	private String refresh(String label) {
+	public String refresh(String label) {
```

Comment:
Fixed. It was a temporary idea to use that method in the health check which was not a good idea :)